app/common/src/map/PriceMarker.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+
+import Color from '../Color';
+
+const styles = StyleSheet.create({
+  container: {
+    // price marker is wider than default, so needs to be shifted more to the left
+    marginLeft: -30,
+    marginTop: -10,
+  },
+  bubble: {
+    flexDirection: 'row',
+    paddingVertical: 5,
+    paddingHorizontal: 8,
+    backgroundColor: '#fff',
+    borderRadius: 3,
+    borderColor: '#fff',
+    borderWidth: 2,
+  },
+  price: {
+    color: '#000',
+    fontSize: 18,
+  },
+  currency: {
+    color: Color.grey._700,
+    fontSize: 15,
+  },
+  arrowBorder: {
+    backgroundColor: 'transparent',
+    borderColor: 'transparent',
+    borderWidth: 8,
+    borderTopColor: '#ffffff',
+    alignSelf: 'center',
+  },
+  arrow: {
+    backgroundColor: 'transparent',
+    borderColor: 'transparent',
+    borderWidth: 7,
+    borderTopColor: '#fff',
+    alignSelf: 'center',
+    marginTop: -18,
+  },
+});
+
+const selectedStyles = StyleSheet.create({
```

Comment:
My idea was to use `StyleSheet.flatten` or spread operator or `_.defaultsDeep` before using it in the `render` method because then you can do just:

```js
<View style={styles.bubble}> // <<< Just "styles" because we already have final style...
  <Text> // <<< Uhm, why do we need this component here?
    <Text style={styles.price}>
      {props.price}
      // ...
```

Reply:
ahh, ok, thanks for idea

wrapping text is necessary for multiple text elements with various heighs, like in this case, so the smaller text is not flying in the air and it's aligned to the baseline